### DISCUSSION TITLE
{context: 'core'} {context: 'core/relayer'} Error: WebSocket

### DISCUSSION THREAD

**Discussion by ebennaga** [WalletConnect/walletconnect-monorepo#3612] [Developer Support]
I got bug from console. what should i do to fix it?

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/2337413/81ad4e21-7446-4837-83d4-cbd3445f5520)

**Comment by glitch-txs** [comment]
Hi, is this on localhost? could you share a [MRE](https://github.com/orgs/WalletConnect/discussions/3457) with more info? Thanks 🙏

**Reply by ebennaga** [reply]
yes,this is localhost.

this issues is similiar with https://github.com/wagmi-dev/references/issues/427

**Reply by glitch-txs** [reply]
can you try generating a new project id?

**Comment by ebennaga** [comment]
Sure, I did but it doesn't work. that's my code

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/2337413/0276e1ee-5a75-4884-9691-f32b2c427268)
